Assets/00_Content/Scripts/UI/HUD.cs

```diff
@@ -19,12 +31,28 @@ public class HUD : MonoBehaviour {
 	}
 
 	void Update() {
-		if (RoundController.Instance != null)
-			roundTimerText.text = Mathf.RoundToInt(RoundController.Instance.RoundTimer).ToString();
+		UpdateClock();
+	}
+
+	private void UpdateClock() {
+		if (RoundController.Instance != null) {
+			float progress = RoundController.Instance.RoundTimer / RoundController.Instance.RoundDuration;
+
+			clockBackgroundDark.fillAmount = 1 - progress;
+			clockPointer.localEulerAngles = new Vector3(0, 0, -MathX.RemapClamped(progress, 0, 1, 0, 360));
```

Comment:
And then you will be able to replace this by the much simpler `-progress * 360`.